### ISSUE TITLE
fails if $LANG env var is undefined

### ISSUE THREAD

**Issue by laur89** [ianka/xfs_undelete#14]
```
./xfs_undelete -t 2020-08-30 -o /mnt/disks/flashdrive /dev/sdc1 
Starting recovery.
no such variable805599295 ( 98%)
    (read trace on "::env(LANG)")
    invoked from within
"set lang $::env(LANG)"
    (procedure "dd" line 5)
    invoked from within
"dd if=$::fs of=$of bs=$::blocksize skip=[dict get $extents 0 skip] seek=0 count=1 conv=notrunc status=none"
    (procedure "investigateInodeBlock" line 80)
    invoked from within
"investigateInodeBlock $ag $iblock"
    (procedure "traverseInodeTree" line 40)
    invoked from within
"traverseInodeTree $ag $agi_root"
    ("for" body line 10)
    invoked from within
"for {set ag 0} {$ag<$agcount} {incr ag} {
        ## Read inode B+tree information sector of this allocation group.
        seek $fd [expr {$blocksize*$agblocks*$ag..."
    (file "./xfs_undelete" line 577)
```

Running on unraid (a Slackware derivative)

tcl-8.6.10
tcllib-1.20

`/dev/sdc` is unmounted

**Comment by laur89** [comment]
Happens when `LANG` env var is not defined.
Perhaps default to `en_US.UTF-8` when undefined?

**Comment by ianka** [comment]
Default should be LANG=C. I check it.

**Comment by ianka** [comment]
Fixed with [commit 95858a1de97f4a37e36818be6944cc6c57db3338](https://github.com/ianka/xfs_undelete/commit/95858a1de97f4a37e36818be6944cc6c57db3338)
